### ISSUE TITLE
CheckBox & RadioButton ContentPresenter missing

### ISSUE THREAD

**Issue by robertmuehsig** [firstfloorsoftware/mui#78]
I try to get TextWrapping working for RadioButton & CheckBox "Labels". 
Without any styling:
I can do this kind of stuff:

```
    <RadioButton Grid.Column="0" Grid.Row="0" Content="Very very very very Very very very very Very very very very Very very very very Very very very very long" />

    <RadioButton Grid.Column="0" Grid.Row="1">
            <TextBlock TextWrapping="Wrap">Very very very very Very very very very Very very very very Very very very very Very very very very long</TextBlock>
    </RadioButton> 
```

Results in:
![image](https://cloud.githubusercontent.com/assets/756703/11657838/aaadf70c-9dbf-11e5-85c1-5ce1df7625dd.png)

When I apply the ModernUI style I get this:
![image](https://cloud.githubusercontent.com/assets/756703/11657856/c243207c-9dbf-11e5-9eaf-ee8a94fae9a5.png)

The same "issue" exists for checkboxes as well.

I guess the reason is here:
https://github.com/firstfloorsoftware/mui/blob/master/1.0/FirstFloor.ModernUI/Shared/Assets/CheckBox.xaml#L20
https://github.com/firstfloorsoftware/mui/blob/master/1.0/FirstFloor.ModernUI/Shared/Assets/RadioButton.xaml#L13

Currently I couldn't find a workaround.

Full demo code:
```
<Window x:Class="WpfApplication9.MainWindow"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:WpfApplication9"
        mc:Ignorable="d"
        Title="MainWindow" Height="550" Width="525">
    <Window.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <ResourceDictionary Source="/FirstFloor.ModernUI;component/Assets/ModernUI.xaml" />
                <ResourceDictionary Source="/FirstFloor.ModernUI;component/Assets/ModernUI.Light.xaml" />
            </ResourceDictionary.MergedDictionaries>
        </ResourceDictionary>
    </Window.Resources>
    <Grid x:Name="test">
        <Grid.ColumnDefinitions>
            <ColumnDefinition Width="200"></ColumnDefinition>
        </Grid.ColumnDefinitions>
        <Grid.RowDefinitions>
            <RowDefinition></RowDefinition>
            <RowDefinition></RowDefinition>
            <RowDefinition></RowDefinition>

        </Grid.RowDefinitions>
        <RadioButton Grid.Column="0" Grid.Row="0" Content="Very very very very Very very very very Very very very very Very very very very Very very very very long" />

        <RadioButton Grid.Column="0" Grid.Row="1">
            <TextBlock TextWrapping="Wrap">Very very very very Very very very very Very very very very Very very very very Very very very very long</TextBlock>
        </RadioButton> 
    </Grid>
</Window>
```

PR welcome or is this project "done"?

**Comment by robertmuehsig** [comment]
With the changes from this PR https://github.com/firstfloorsoftware/mui/pull/82 it seems to work:
![image](https://cloud.githubusercontent.com/assets/756703/11972890/f2541dcc-a952-11e5-92dd-3f157d329f86.png)

**Comment by robertmuehsig** [comment]
Fixed in 1.0.8. Thanks!
